internal/e2e/plugin_perf_test.go

```diff
@@ -101,13 +99,13 @@ func BenchmarkPluginFilter(b *testing.B) {
 
 					b.ResetTimer()
 					for i := 0; i < b.N; i++ {
-						// Intentionally change the pointer to simulate a new scheduling cycle.
+						// Intentionally change the pointer and run PreFilter,
```

Comment:
we don't need to change the pointer anymore right?
https://github.com/kubernetes-sigs/kube-scheduler-wasm-extension/pull/42

Reply:
agreed and done!